Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendAllPANWIoTDevicesToCiscoISE/SendAllPANWIoTDevicesToCiscoISE.py

```diff
@@ -0,0 +1,217 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+cisco_ise_field_map = {
+    "ip": ["ZingboxIpAddress", "PanwIoTIpAddress"],
+    "ip address": ["ZingboxIP", "PanwIoTIP"],
+    "ip_address": ["ZingboxIP", "PanwIoTIP"],
+    "profile": ["ZingboxProfile", "PanwIoTProfile"],
+    "category": ["ZingboxCategory", "PanwIoTCategory"],
+    "risk_score": ["ZingboxRiskScore", "PanwIoTRiskScore"],
+    "risk score": ["ZingboxRiskScore", "PanwIoTRiskScore"],
+    "confidence": ["ZingboxConfidence", "PanwIoTConfidence"],
+    "confidence score": ["ZingboxConfidence", "PanwIoTConfidence"],
+    "confidence_score": ["ZingboxConfidence", "PanwIoTConfidence"],
+    "tag": ["ZingboxTag", "PanwIoTTag"],
+    "asset_tag": ["ZingboxTag", "PanwIoTTag"],
+    "Tags": ["ZingboxTag", "PanwIoTTag"],
+    "hostname": ["ZingboxHostname", "PanwIoTHostname"],
+    "osCombined": ["ZingboxOS", "PanwIoTOS"],
+    "model": ["ZingboxModel", "PanwIoTModel"],
+    "vendor": ["ZingboxVendor", "PanwIoTVendor"],
+    "Serial Number": ["ZingboxSerial", "PanwIoTSerial"],
+    "Serial_Number": ["ZingboxSerial", "PanwIoTSerial"],
+    "endpoint protection": ["ZingboxEPP", "PanwIoTEPP"],
+    "endpoint_protection": ["ZingboxEPP", "PanwIoTEPP"],
+    "AET": ["ZingboxAET", "PanwIoTAET"],
+    "External Network": ["ZingboxInternetAccess", "PanwIoTInternetAccess"],
+    # "last activity": "ZingboxLastActivity"
+}
+'''
+Get the error message or active cisco ise instance
+'''
+
+
+def get_active_ise_instance():
+    # get active Cisco ISE instance
+    response = demisto.executeCommand("GetCiscoISEActiveInstance", {})
+    err_msg = None
+    active_instance = None
+
+    data = response[0].get('EntryContext', {})
+
+    if 'PaloAltoIoTIntegrationBase.ActiveNodeInstance' in data:
+        active_instance = data.get('PaloAltoIoTIntegrationBase.ActiveNodeInstance')
+    elif 'PaloAltoIoTIntegrationBase.NodeErrorStatus' in data:
+        err_msg = data.get('PaloAltoIoTIntegrationBase.NodeErrorStatus')
+
+    return active_instance, err_msg
+
+
+'''
+Extract any connection error or error code if possible,
+Otherwise just return the original error
```

Comment:
please consider refactoring with a `main()` entry point, it is quite hard to track this code.

Reply:
added main()